### PR TITLE
docs: ✏️ add Alexjs github action to README

### PR DESCRIPTION
**Pull request by theashraf**
catch insensitive, inconsiderate writing in your markdown docs using
Alexjs Github action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -404,7 +406,7 @@ https://github.com/sdras/awesome-actions/workflows/Lint%20Awesome%20List/badge.s
 - [Continuous deployment of Next.js app with Up](https://medium.com/@romanenko/simple-ci-for-next-js-projects-with-apex-up-github-actions-6f0b1b9a5400)
 - [Converting Docker-based Actions to JavaScript/TypeScript](https://httgp.com/converting-github-actions-from-docker-to-javascript/)
 - [GitHub Actions CI for Swift/iOS Projects](https://medium.com/rosberryapps/github-actions-ci-for-swift-projects-c129baceed1a)
-> Please don't hesitate to make a PR if you have more resources to share. Check out [contributing.md](contributing.md) for more information
+  > Please don't hesitate to make a PR if you have more resources to share. Check out [contributing.md](contributing.md) for more information
```
Comment (peaceiris) [inline README.md]: This indent seems to be unnecessary.

Comment (theashraf) [reply README.md]: Sorry, this indent was made by my vscode prettier plugin.
should I remove it?

Comment (peaceiris) [reply README.md]: I got it. Please add a new line above this line. Your prettier will not indent this line.

Comment (theashraf) [reply README.md]: @peaceiris done!

**Comment by gep13** [comment]
@theashraf your changes have been merged, thanks for your contribution 👍

**Pull request merged**